composer.json

```diff
@@ -9,16 +9,16 @@
         "source": "https://github.com/OSSIndex/bach"
     },
     "require": {
-        "php": ">=7.1.3 || <9.0",
+        "php": "^7.1.3 || ^8.0",
```

Comment:
Interesting. I didn't realize the difference between `>=` and `^` could cause this sort of problem.
The fix looks good to me.